### ISSUE TITLE
[BITV]: Gap of keyboard navigation through a toolbar

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/text#5222]
1. Open a text file and navigate via keyboard
2. Focus on a "Close" button
3. Make 1 next navigation stop (via tab) and see: there is no element which is focused 
4. Push Enter
5. Dialog to open a file will be open -> completely unexpected behavior

Please set a right button and focus on it that there will be no gap in between

![Peek 2024-01-10 10-42](https://github.com/nextcloud/text/assets/6078378/c9e206d2-1a56-4295-adf6-637855f1cf3b)

Same behavior is on a preview page:
![Screenshot from 2024-01-10 11-32-57](https://github.com/nextcloud/text/assets/6078378/f2d2f0c0-9d2f-48f1-b1e0-4941a413d1b4)

**Comment by JuliaKirschenheuter** [comment]
backport to 28 is still open

**Comment by juliushaertl** [comment]
No https://github.com/nextcloud/text/pull/5227

**Comment by szaimen** [comment]
I am not sure what is happening here but something seems to be very broken:
### cycling through with tab:

https://github.com/nextcloud/text/assets/42591237/a73ac051-6933-4382-8cfd-54797f1e075a


### cycling through with shift-tab:

https://github.com/nextcloud/text/assets/42591237/4be253e0-cc4e-4966-a589-44c83c416520

**Comment by szaimen** [comment]
This doesnt seem to work correclty yet on stable28:

https://github.com/nextcloud/text/assets/42591237/ec77c856-0c2b-41fc-a38e-96a322344db6

(there is a focus-gap or it seems to focus elements that are not visible)
